### PR TITLE
Requirizing workers

### PR DESCRIPTION
**Pull request by iprotsyuk**
Image loading isn't fixed yet because it's implemented via a worker-like object so some specific problems are there. Other workers work fine.

### CODE REVIEW

File: js/Workspace.js [hunk 1]
```diff
@@ -353,24 +353,30 @@ function(ColorMap, EventSource, Scene2D, Scene3D, THREE) {
...
-                            addError('Operation failed: ' + event.data.message);
-                        } else if (event.data.status == 'working') {
-                            setStatus(event.data.message);
-                        }
+                        switch (event.data.status) {
+                            case 'completed':
+                                setStatus('');
+                                resolve(event.data);
+                                task.cancel();
+                                console.info('Task ' + taskType.key + ' completed in ' +
```
Comment (antgonza) [inline js/Workspace.js:366]: is this OK to leave?

File: js/workers/Raycaster.js [hunk 2]
```diff
@@ -55,6 +57,7 @@ function(require, THREE) {
         console.log('Raycast time: ' +
```
Comment (antgonza) [inline js/workers/Raycaster.js:60]: Same here, do we want to leave the console.log?

**Comment by antgonza** [comment]
2 comments, after that 👍, sooo close!

**Comment by iprotsyuk** [comment]
@antgonza, those log messages just indicate execution time. I guess it might be helpful to quickly evaluate performance if that modules will be changed in future. Or you mean something different?

**Comment by antgonza** [comment]
OK, just wanted to be sure that they weren't left by mistake.

**Comment by antgonza** [comment]
Cool, thanks! 👍 @ElDeveloper, could you take a look?

**Comment by ElDeveloper** [comment]
👍

**Pull request merged**